src/components/notebook-utils.js

```diff
@@ -90,6 +90,9 @@ export const getAppType = label => _.find(tool => tool.label === label)(tools)?.
 // Does the tool label correspond to an app?
 export const isToolAnApp = label => getAppType(label) !== undefined
 
+// Returns registered appTypes.
+export const getAllAppTypes = _.flow(_.map('appType'), _.compact)(tools)
```

Comment:
This seems to be a constant. Can we name it accordingly?
```suggestion
export const allAppTypes = _.flow(_.map('appType'), _.compact)(tools)
```

Reply:
I thought you wanted the get prefix. I'm personally used to all caps for constants… but that's not our convention here, correct? I feel like I'm getting confused about how things should be named. Is "get" and "is" reserved for functions/methods?

Reply:
(I did make the suggested change, but would still like clarification on our naming standards)

Reply:
> I did make the suggested change, but would still like clarification on our naming standards

I would too. See https://github.com/DataBiosphere/terra-ui/pull/2702#discussion_r762449605.